ios/PriceWidget/Models/CoinGeckoToken.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  CoinGeckoToken.swift
+//  Rainbow
+//
+//  Created by Ben Goldberg on 11/2/21.
+//  Copyright © 2021 Facebook. All rights reserved.
+//
+
+import Foundation
+
+@available(iOS 14.0, *)
+struct CoinGeckoToken: Codable {
+  struct Platform: Codable {
+    let ethereum: String
```

Comment:
I can't remove this because the CoinGecko API returns tokens for all asset platforms. The response is filtered though so only the Rainbow-eligible ones are used for the widget.